File: sdks/thunkmetrc-go/wrapper/models/sales_update_sale_receipts_request_item.go
package models

type SalesUpdateSaleReceiptsRequestItem struct {
    CaregiverLicenseNumber string `json:"CaregiverLicenseNumber,omitempty"`
    ExternalReceiptNumber string `json:"ExternalReceiptNumber,omitempty"`
    Id int `json:"Id,omitempty"`
    IdentificationMethod string `json:"IdentificationMethod,omitempty"`
    PatientLicenseNumber string `json:"PatientLicenseNumber,omitempty"`
    PatientRegistrationLocationId int `json:"PatientRegistrationLocationId,omitempty"`
    SalesCustomerType string `json:"SalesCustomerType,omitempty"`
    SalesDateTime string `json:"SalesDateTime,omitempty"`
    Transactions []*SalesUpdateSaleReceiptsRequestItemTransaction `json:"Transactions,omitempty"`
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/SalesUpdateSaleReceiptsRequestItem.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class SalesUpdateSaleReceiptsRequestItem(
    @JsonProperty("caregiverLicenseNumber")
    val caregiverLicenseNumber: String? = null,
    @JsonProperty("externalReceiptNumber")
    val externalReceiptNumber: String? = null,
    @JsonProperty("id")
    val id: Int? = null,
    @JsonProperty("identificationMethod")
    val identificationMethod: String? = null,
    @JsonProperty("patientLicenseNumber")
    val patientLicenseNumber: String? = null,
    @JsonProperty("patientRegistrationLocationId")
    val patientRegistrationLocationId: Int? = null,
    @JsonProperty("salesCustomerType")
    val salesCustomerType: String? = null,
    @JsonProperty("salesDateTime")
    val salesDateTime: String? = null,
    @JsonProperty("transactions")
    val transactions: List<SalesUpdateSaleReceiptsRequestItemTransactions>? = null
)
